libcontainer/utils/utils.go

```diff
@@ -165,3 +165,22 @@ func Annotations(labels []string) (bundle string, userAnnotations map[string]str
 	}
 	return
 }
+
+// DeepCopy makes a deep copy from src into dst.
+func DeepCopy(dst interface{}, src interface{}) error {
```

Comment:
The `dst` `src` order of arguments confused me for a bit, but I always have the same with golang's builtin `copy()`, so 🤷‍♂️ 😅